lib/source_file.h

```diff
@@ -56,6 +56,7 @@ class SourcePosition final {
               columnNumber(0) {}
 
     SourcePosition(unsigned lineNumber, unsigned columnNumber);
+    SourcePosition& operator=(const SourcePosition&) = default;
```

Comment:
I noticed the same issue in a few other places: See: **ProgramPoint** and **Node**.

Reply:
Do you get warnings about them from gcc?